Gemfile.lock

```diff
@@ -281,14 +281,15 @@ GEM
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webrick (1.8.1)
+    webrick (1.8.2)
     websocket-driver (0.7.6)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    zeitwerk (2.6.17)
+    zeitwerk (2.7.1)
 
 PLATFORMS
   x86_64-darwin-19
+  x86_64-darwin-21
```

Comment:
Undo gemfile.lock related changes too. They should not be part of this PR.

Reply:
done.